test/simperium/mock_bucket_store.js

```diff
@@ -15,10 +15,10 @@ BucketStore.prototype.get = function( id, callback ) {
 	} );
 };
 
-BucketStore.prototype.update = function( id, object, callback ) {
+BucketStore.prototype.update = function( id, object, isIndexing, callback ) {
```

Comment:
I think the entire project could use another look with an es6 lens.

Originally it was not being transpiled and the biggest changes after adding babel was the es6 `import` syntax.